Formula/prql-compiler.rb

```diff
@@ -1,19 +1,17 @@
 class PrqlCompiler < Formula
-  desc "PRQL is a modern language for transforming data — a simpler and more powerful SQL"
+  desc "PRQL is a modern language for transforming data: a simpler and more powerful SQL"
```

Comment:
Not a huge deal, but the existing line is a canonical sentence we use in lots of places; do we have a strong reason to change it? If so, we should change it everywhere...

Reply:
I only changed it to avoid hitting the length limit Homebrew prefers for descriptions.  Is there a better way to do that you'd prefer?

Reply:
Oh interesting! That's fine then. I guess we could shorten "language" to "lang" but I probably prefer your change